### ISSUE TITLE
quarkus-vertx-web hangs if port is in use

### ISSUE THREAD

**Issue by jiekang** [quarkusio/quarkus#2544]
**Describe the bug**

When starting a quarkus vertx web app that listens on a port that is in use, the quarkus process hangs and does not respond to SIGTERM. 

**Expected behavior**

It would be acceptable for the application to exit with error message informing that the port is in use.

**Actual behavior**

The application hangs with no logs and does not respond to SIGTERM.

**To Reproduce**
1. Start an http service listening on port 8080
2. Start a quarkus-vertx-web app listening on port 8080 (default). This was tried in JVM and Native modes.
3. Try to stop the quarkus-vertx-web app via SIGTERM.

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: 
```
Linux <host-redacted> 5.0.16-200.fc29.x86_64 #1 SMP Tue May 14 18:27:35 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```
 - GraalVM version (if different from Java): 
```
graalvm-ce-1.0.0-rc16
```
 - Quarkus version or git rev: 
```
<dependency>
    <groupId>io.quarkus</groupId>
    <artifactId>quarkus-vertx-web</artifactId>
    <version>0.15.0</version>
</dependency>

```

**Comment by cescoffier** [comment]
@jiekang I've fixed it in #2552.

**Comment by jiekang** [comment]
Looks good, thanks!
